packages/liquidator/src/liquidator.js

```diff
@@ -59,6 +61,7 @@ class Liquidator {
     this.empMinSponsorSize = empProps.minSponsorSize;
 
     this.empIdentifier = empProps.priceIdentifier;
+    this.empLiquidationLiveness = empProps.LiquidationLiveness;
```

Comment:
two points on this:
1) `LiquidationLiveness` should properly be called `liquidationLiveness`.
2) if you using a new property on `empProps` then you will need to update the `empProps` object that is fed into `Liquidator` on construction from the `index.js` entry point [here](https://github.com/UMAprotocol/protocol/blob/04c3c27d969c3cc7edf28317ef4398bb7b7bb407/packages/liquidator/index.js#L101).